addons/stock/models/stock_move.py

```diff
@@ -578,6 +578,7 @@ def _merge_moves(self, merge_into=False):
         moves_to_merge = []
         for candidate_moves in candidate_moves_list:
```

Comment:
normally better speed, no ?
`for candidate_moves in candidate_moves_list.with_context(prefetch_fields=False)`

Reply:
`candidate_moves_list` is a list, not a recordset